cfgov/v1/models/learn_page.py

```diff
@@ -106,6 +107,47 @@ class DocumentDetailPage(AbstractFilterPage):
     template = 'document-detail/index.html'
 
 
+class StoryPage(AbstractFilterPage):
```

Comment:
I don't think that EHAS is significantly different enough from other types of content that it would warrant its own app, but I'm not entirely sure what our philosophy is on when things should be turned into separate apps.